Quiet day (#437)
lsst-dm/dm_dev_guide: team/quiet.rst

############
Quiet Friday
############

> Starting on 2020-09-25 DMLT will commence **Quiet Friday**. 

Reviewer: I suggest not hard-coding the date in the Developer Guide — it won't be relevant in 8 days. Let's just put it in the announcement.

Author: Historic moment!
But ok

Guidelines
==========
The driver for the quiet day is to allow staff a block of time without interruptions or the need to be online and receptive. 

> By reducing traffic we will reduce FOMO on discussion topics. 

Reviewer: Somebody's going to get confused about what “FOMO” means. Suggest rephrasing.

Reviewer 2: Or add it to the list of acronyms. It is very commonly used. (although I agree, someone will get confused)

> Hence there should be: 

Reviewer 3: ```suggestion
Hence: 
```

> - No general meetings should be planned. There may of course be exceptions, for example if your focused work includes a co-work with others who agree to meet this is ok. Such agreements should be between peers and not use any hierarchy. You may elect to attend meetings/workshops outside DM - or occasional we may be required to attend such meetings.

Reviewer 4: We prefer one line per sentence since that makes commenting easier and github happier.

occasional -> occasionally

> - No general meetings should be planned. There may of course be exceptions, for example if your focused work includes a co-work with others who agree to meet this is ok. Such agreements should be between peers and not use any hierarchy. You may elect to attend meetings/workshops outside DM - or occasional we may be required to attend such meetings.

Reviewer 3: ```suggestion
- No general meetings should be planned. There may of course be exceptions, for example if your focused work includes a co-work with others who agree to meet, this is OK. Such agreements should be between peers and not use any hierarchy. You may elect to attend meetings/workshops outside DM — or occasionally we may be required to attend such meetings.
```

> - No general meetings should be planned. There may of course be exceptions, for example if your focused work includes a co-work with others who agree to meet this is ok. Such agreements should be between peers and not use any hierarchy. You may elect to attend meetings/workshops outside DM - or occasionally we may be required to attend such meetings.

Reviewer 3: ```suggestion
- No planned meetings, generally. There may of course be exceptions, for example if your focused work includes a co-work with others who agree to meet this is ok. Such agreements should be between peers and not use any hierarchy. You may elect to attend meetings/workshops outside DM - or occasionally we may be required to attend such meetings.
```

> - No Slack messages on general channels. You may slack an individual, or a group of individuals if you have agreed such interactions ahead of time. Conversations should not be carried out on general channels. There should be no expectation of response to Slack messages.

Reviewer 3: ```suggestion
- No Slack messages on general channels. You may slack an individual or a group of individuals if you have agreed such interactions ahead of time. Conversations especially should not be carried out on general channels. There should be no expectation of response to Slack messages.
```

> Asynchronous communication is perfectly acceptable on the understanding than no response is to be expected before the next work day (Monday).

Reviewer 4: "the understanding that"

Author: got it

> Asynchronous communication is perfectly acceptable on the understanding that no response is to be expected before the next work day (Monday).

Reviewer 5: "(normally Monday)" - since there are quite a few Monday holidays in our calendar

Author: ok

> Hence it is ok to, for example:

Reviewer 3: ```suggestion
Hence it is OK to, for example:
```

> Hence it is OK To, for example:

Reviewer 5: "to"

> - Send email.  Though you may consider using a delayed send.

Reviewer 3: ```suggestion
- Send email, though you may consider using a delayed send.
```

> - Update Jira issues, which will result in email notifications that are ok. 

Reviewer 3: ```suggestion
- Update Jira issues, which will result in email notifications that are OK. 
```

> - Perform github work. Code reviews, merges etc. which will result in notifications are ok.

Reviewer 4: GitHub

Author: ok

> - Perform github work. Code reviews, merges etc. which will result in notifications are ok.

Reviewer 3: ```suggestion
- Perform github work. Code reviews, merges etc. which will result in notifications are OK.
```

> - Perform GitHub work. Code reviews, merges etc. which will result in notifications are OK.

Reviewer 6: I think it's probably worth also mentioning confluence

> This will be generally publicized outside DM hence it is ok to set up an out office reply.

Reviewer: Maybe be cautious here? We might tell other parts of Rubin about the quiet day, but they'll (probably) forget, and we can't expect the wider community to be aware of it.

(Arguably, the wider community can't expect us to be instantly responsive, of course.)

> This will be generally publicized outside DM hence it is ok to set up an out office reply.

Reviewer 3: ```suggestion
This will be generally publicized outside DM, hence it is OK to set up an out-of-office reply.
```

Emergencies and on call
=======================

> There may always be an emergency, urgent bug, machine failure, fire, flooding etc. 

Reviewer 3: ```suggestion
There may on occasion be an emergency, urgent bug, machine failure, fire, flooding etc. 
```

> In this case it is ok to make Friday noisy.

Reviewer 3: ```suggestion
In this case it is OK to make Friday noisy.
```
